docs/source/pandas_df_conversion.rst

```diff
@@ -0,0 +1,252 @@
+            "id"
+
+   "9650885788371886430",	"None",	"2",	"1942.600083",	"395.239277",	"1932.484009",	"1950.834351",	"395.239199",	"397.245758",	"0.808494",	"157572000.0", "10.061090", "168"
+   "18416216708636999474",	"None",	"2",	"1749.138335",	"443.711224",	"1735.693115",	"1763.343506",	"443.71112",	"445.717531",	"0.893553",	"54069300.0", "14.156094",	"169"
+
+ConsensusMap
+************
+
+**pyopenms.ConsensusMap.get_df()**
+        Generates a pandas DataFrame with both consensus feature meta data and intensities from each sample.
+
+        **Returns:**
+
+        **pandas.DataFrame** 
+        
+        consensus map meta data and intensity stored in pandas DataFrame
+
+**pyopenms.ConsensusMap.get_intensity_df()**
+        Generates a pandas DataFrame with feature intensities from each sample in long format (over files).
+        For labelled analyses channel intensities will be in one row, therefore resulting in a semi-long/block format.
+        Resulting DataFrame can be joined with result from get_metadata_df by their index 'id'.
+
+        **Returns:**
+
+        **pandas.DataFrame** 
+        
+        intensity DataFrame
+
+**pyopenms.ConsensusMap.get_metadata_df()**
+        Generates a pandas DataFrame with feature meta data (sequence, charge, mz, RT, quality).
+        Resulting DataFrame can be joined with result from get_intensity_df by their index 'id'.
+
+        **Returns:**
+
+        **pandas.DataFrame** 
+        
+        DataFrame with metadata for each feature (such as: best identified sequence, charge, centroid RT/mz, fitting quality)
+
+**Examples:**
+
+.. code-block:: python
+
+    urlretrieve(url+'ConsensusXMLFile_1.consensusXML', 'ConsensusXMLFile_1.consensusXML')
+    consensus_map = ConsensusMap()
+    ConsensusXMLFile().load('ConsensusXMLFile_1.consensusXML', consensus_map)
+
+    df = consensus_map.get_df()
+    df.head(2)
+    
+.. csv-table:: consensus_map.get_df()
+   :widths: 2 10 20 20 20 20 30 30
+   :header:     , "sequence",	"charge",	"RT",	"mz",	"quality",	"data/MapAlignmentFeatureMap1.xml",	"data/MapAlignmentFeatureMap2.xml"
+            "id"
+
+   "0",	"A",	"0",	"1273.27",	"904.470",	"1.1",	"31253900.0",	"0.0"
+   "1",	"E",	"0",	"1248.33",	"897.449",	"1.2",	"25917900.0",	"25917900.0"
+
+.. code-block:: python
+
+    df = consensus_map.get_intensity_df()
+    df.head(2)
+
+.. csv-table:: consensus_map.get_intensity_df()
+   :widths: 2 30 30
+   :header:     , "data/MapAlignmentFeatureMap1.xml",	"data/MapAlignmentFeatureMap2.xml"
+            "id"
+
+   "0",	"31253900.0",	"0.0"
+   "1",	"25917900.0",	"25917900.0"
+
+.. code-block:: python
+
+    df = consensus_map.get_metadata_df()
+    df.head(2)
+
+.. csv-table:: consensus_map.get_metadata_df()
+   :widths: 2 10 20 20 20 20
+   :header:     , "sequence",	"charge",	"RT",	"mz",	"quality"
+            "id"
+
+   "0",	"A",	"0",	"1273.27",	"904.470",	"1.1"
```

Comment:
maybe a file with more realistic entries would be better here (e.g., not sure but maybe from the ProteomicsLFQ testfiles?)